### PR TITLE
Bug 1211789: invalid slugs display error warning during synchronization.

### PR DESCRIPTION
**Pull request by jotes**
Hi, @mathjazz could you r?

### CODE REVIEW

File: pontoon/sync/tests/test_sync_projects.py [hunk 1]
```diff
@@ -69,6 +72,24 @@ def test_no_matching_projects(self):
...
+
+        self.mock_sync_project.delay.assert_called_with(
+            handle_project.pk,
+            ANY,
+            no_pull=False,
+            no_commit=False,
+            force=False
+        )
+
+        assert_equal(self.command.stderr.getvalue(), "Couldn't find projects with following slugs: aaa, bbb")
```
Comment (mathjazz) [inline pontoon/sync/tests/test_sync_projects.py]: Nit: use single quotes.

File: pontoon/sync/management/commands/sync_projects.py [hunk 2]
```diff
@@ -49,6 +49,10 @@ def handle(self, *args, **options):
         if len(projects) < 1:
             raise CommandError('No matching projects found.')
 
+        if len(projects) != len(args):
+            invalid_slugs = sorted(set(args).difference(set(projects.values_list('slug', flat=True))))
+            self.stderr.write("Couldn't find projects with following slugs: {}".format(', '.join(invalid_slugs)))
```
Comment (mathjazz) [inline pontoon/sync/management/commands/sync_projects.py]: Nit: use single quotes.

**Comment by mathjazz** [comment]
Good job, r+.

**Comment by jotes** [comment]
@mathjazz fixed! :tada: :dancers:

**Pull request merged**